### PR TITLE
Add gfflu v0.0.1

### PR DESCRIPTION
**Pull request by peterk87**
This PR adds a Conda recipe for gfflu v0.0.1 a Python CLI app to generate annotations of Influenza A virus (IAV) gene segment nucleotide sequences with [BLASTX](https://blast.ncbi.nlm.nih.gov/Blast.cgi?PROGRAM=blastx&PAGE_TYPE=BlastSearch&LINK_LOC=blasthome) and [Miniprot](https://github.com/lh3/miniprot) using the same protein sequences as [Influenza Virus Sequence Annotation Tool](https://www.ncbi.nlm.nih.gov/genomes/FLU/annotation) and output a [GFF3](https://github.com/The-Sequence-Ontology/Specifications/blob/master/gff3.md) file with the expected genetic features for each of the 8 IAV gene segments.

----

Please read the [guidelines for Bioconda recipes](https://bioconda.github.io/contributor/guidelines.html) before opening a pull request (PR).

* If this PR adds or updates a recipe, use "Add" or "Update" appropriately as the first word in its title.
* New recipes not directly relevant to the biological sciences need to be submitted to the [conda-forge channel](https://conda-forge.org/docs/) instead of Bioconda.
* PRs require reviews prior to being merged. Once your PR is passing tests and ready to be merged, please issue the `@BiocondaBot please add label` command.
* Please post questions [on Gitter](https://gitter.im/bioconda/Lobby) or ping `@bioconda/core` in a comment.

<details>
  <summary>Please use the following BiocondaBot commands:</summary>

Everyone has access to the following BiocondaBot commands, which can be given in a comment:

<table>
  <tr>
    <td><code>@BiocondaBot please update</code></td>
    <td>Merge the master branch into a PR.</td>
  </tr>
  <tr>
    <td><code>@BiocondaBot please add label</code></td>
    <td>Add the <code>please review & merge</code> label.</td>
  </tr>
  <tr>
    <td><code>@BiocondaBot please fetch artifacts</code></td>
    <td>Post links to CI-built packages/containers. <br />You can use this to test packages locally.</td>
  </tr>
</table>

For members of the Bioconda project, the following command is also available:

<table>
  <tr>
    <td><code>@BiocondaBot please merge</code></td>
    <td>Upload built packages/containers and merge a PR. <br />Someone must approve a PR first! <br />This reduces CI build time by reusing built artifacts.</td>
  </tr>
</table>

Also, the bot watches for comments from non-members that include `@bioconda/<team>` and will automatically re-post them to notify the addressed `<team>`.

</details>

### CODE REVIEW

**Comment by peterk87** [comment]
@BiocondaBot please add label

File: recipes/gfflu/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+requirements:
+  host:
+    - python >=3.8
+    - pip
+    - hatch
+  run:
+    - blast
+    - miniprot
+    - python >=3.8
+    - pip
```
Comment (siebrenf) [inline recipes/gfflu/meta.yaml:26]: pretty sure you dont need `pip` in the `run` section

File: recipes/gfflu/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,41 @@
+{% set name = "gfflu" %}
+{% set version = "0.0.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name | replace("_", "-") }}/{{ name }}-{{ version }}.tar.gz
```
Comment (siebrenf) [inline recipes/gfflu/meta.yaml:9]: ` | replace("_", "-")` doesn't do anything

**Review by siebrenf — APPROVED** [review]
Unless you use `pip` in your code you can remove it from the `run` section. The recipe looks good otherwise!

File: recipes/gfflu/meta.yaml (hunk 1 above)
Comment (peterk87) [reply recipes/gfflu/meta.yaml]: Thanks! You're right. Removed in 5c7f1d8067

File: recipes/gfflu/meta.yaml (hunk 2 above)
Comment (peterk87) [reply recipes/gfflu/meta.yaml]: Good catch! Removed in 5c7f1d8067

**Comment by peterk87** [comment]
> Unless you use `pip` in your code you can remove it from the `run` section. The recipe looks good otherwise!

Thanks @siebrenf for the review! I've made the changes you suggested in 5c7f1d8067. Please let me know if you have any other suggestions!

**Comment by siebrenf** [comment]
No problem. The recipe seems complete and without unneeded dependencies!

**Pull request merged**